### DISCUSSION TITLE
Centralized text for editor strings

### DISCUSSION THREAD

**Discussion by AMZN-Liv** [o3de/rfcs#29] [Ideas]
**Is your feature request related to a problem? Please describe.**
Today, strings that are surfaced in the editor are not centralized. This means that changing or updating a string (for example, to localize the editor into another language) many files have to be updated separately. 

**Describe the solution you'd like**
A centralized system with a string table for editor error messages and other editor strings will be easier to make changes to text within the editor, and to localize the editor to other languages.

**Comment by sptramer** [comment]
The best implementation for this, and one that the docs team has discussed with UX in the past, is to use [Qt resources](https://doc.qt.io/qt-5/resources.html) to provide a `.strings` file, which could also be used for localization of the Editor.
